addons/packages/vsphere-cpi/1.23.0-alpha.1/bundle/config/values.yaml

```diff
@@ -38,3 +39,9 @@ vsphereCPI:
   http_proxy: ""
   https_proxy: ""
   no_proxy: ""
+  clusterAPIVersion: "run.tanzu.vmware.com/v1alpha2"
+  clusterKind: "TanzuKubernetesCluster"
+  clusterName: "test-cluster-e2e-script"
```

Comment:
Why default the cluster name and UID to a specific value?

Reply:
changed to empty values